packages/d/date/xmake.lua

```diff
@@ -4,10 +4,12 @@ package("date")
     set_description("A date and time library for use with C++11 and C++14.")
     set_license("MIT")
 
-    add_urls("https://github.com/HowardHinnant/date.git")
-
-    add_versions("2024.05.14", "1ead6715dec030d340a316c927c877a3c4e5a00c")
-    add_versions("2021.04.17", "6e921e1b1d21e84a5c82416ba7ecd98e33a436d0")
+    add_urls("https://github.com/HowardHinnant/date/archive/refs/tags/v$(version).tar.gz", {alias = "release"})
+    add_urls("https://github.com/HowardHinnant/date.git", {alias = "git"})
+    
+    add_versions("release:3.0.3", "30de45a34a2605cca33a993a9ea54e8f140f23b1caf1acf3c2fd436c42c7d942")
```

Comment:
Please use a unified version format. Since there is a release version, you can remove the date version.

Reply:
OK